staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -107,6 +107,15 @@ func MonitorRequest(request *http.Request, verb, resource, subresource, scope, c
 		}
 	}
 
+	// make get pod log with follow option api request reported as watch
+	if verb == "GET" && resource == "pods" && subresource == "log" {
```

Comment:
not sure about the data model we have here. Would http_query_watch, http_query_follow, http_queury_detail make sense?